### DISCUSSION TITLE
Could use a global option to disable radarr and sonarr but allow git collections to still create collections

### DISCUSSION THREAD

**Discussion by westy** [meisnate12/Plex-Meta-Manager#554] [Ideas]
Hi,

I'm using the excellent movies collection I have found at OhMyBahGosh/Movies
The problem is, I do not want to have PMM add things to Radarr and Sonarr, and these collections are configured to do this.

I have tried configuring without a token but that still shows up as disabled, so any collections that are configured to add to radarr fail.

Is there an option to override config for a git collection, or could we have one perhaps?
Failing that add an option to radarr and sonarr to configure but disable or something?

Thoughts?

Cheers

**Comment by kevbentz** [comment]
> Hi,
> 
> I'm using the excellent movies collection I have found at OhMyBahGosh/Movies The problem is, I do not want to have PMM add things to Radarr and Sonarr, and these collections are configured to do this.

Change all the "radarr_add: true" and "sonarr_add: true" to "radarr_add: false" and "sonarr_add: false" and it won't add them.

**Reply by westy** [reply]
Yeah, that's what I ended up doing, after taking a local copy of the config.
Kinda defeats the object of using someone elses config though.

Cheers

**Reply by meisnate12** [reply]
yea but each one is uploaded by that user its their choice on how they want it. Thats why in all the default configs that options is not set

**Reply by westy** [reply]
Yeah, I appreciate that. That's why I wondered whether should be able to override at the point where you reference it.
I think having it not break when config is not set up is a simpler solution though, and will work fine.

I'll try it later. Thanks again!

**Comment by meisnate12** [comment]
i think i updated PMM so the colleitons wont fail completely when a radarr/sonarr error occurs let me know if thats no longer the case. (Make sure youre using the late4st develop branch)

and like kevbentz said download and make a local copy of the file and remove the radarr_add and sonarr_add

**Reply by westy** [reply]
Cool thanks, I'll try and have a look over the coming days and let you know.

**Reply by westy** [reply]
Hi,
Sorry for the delay in looking at this.

Is there a way of getting the unraid docker to use the development branch?
Either that, or when will it be coming to the release branch I guess?

Cheers
